### PR TITLE
Adds capability to use fixed height accordions

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
Method initCollapseHeights added to Patternfly javascript library.

### CODE REVIEW

**Comment by dtaylor113** [comment]
LGTM

**Comment by rhamilto** [comment]
Does this mean #178 should be closed without merging?

**Comment by jeff-phillips-18** [comment]
I have killed #178

**Comment by rhamilto** [comment]
LGTM.  @andresgalante?

File: tests-src/accordions.html [hunk 1]
```diff
@@ -4,47 +4,173 @@
...
+    <div class="panel-group" id="accordion">
+      <div class="panel panel-default">
+        <div class="panel-heading">
+          <h4 class="panel-title">
+            <a data-toggle="collapse" data-parent="#accordion" href="#collapseOne">
+              Lorem ipsum
+            </a>
+          </h4>
+        </div>
+        <div id="collapseOne" class="panel-collapse collapse in">
```
Comment (andresgalante) [inline tests-src/accordions.html:19]: @jeff-phillips-18 I don't know how we handle js, but if these are our classes and not bootstrap we should prefix them

File: tests-src/accordions.html [hunk 2]
```diff
@@ -4,47 +4,173 @@
...
+          <div class="panel-body">
+            Curabitur nisl quam, interdum a venenatis a, consequat a ligula. Nunc nec lorem in erat rhoncus lacinia at ac orci. Sed nec augue congue, vehicula justo quis, venenatis turpis. Nunc quis consectetur purus. Nam vitae viverra lacus. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum eu augue felis. Maecenas in dignissim purus, quis pulvinar lectus. Vivamus euismod ultrices diam, in mattis nibh.
+          </div>
+        </div>
+      </div>
+    </div>
+  </div><!--/col-->
+  <div class="col-md-4 col-sm-6">
+    <h2>Fixed Height</h2>
+    <div class="panel-group" id="fixed-accordion" style="height: 350px;">
```
Comment (andresgalante) [inline tests-src/accordions.html:57]: @jeff-phillips-18 Again, this might be my js ignorance. Do we need a height inline style?

**Comment by andresgalante** [comment]
Should it be hole header a clicable area instead of just the name? what do you think @jeff-phillips-18 @dtaylor113 @rhamilto ?

File: tests-src/accordions.html (hunk 1 above)
Comment (jeff-phillips-18) [inline tests-src/accordions.html:19]: These are straight bootstrap classes.

File: tests-src/accordions.html (hunk 2 above)
Comment (jeff-phillips-18) [inline tests-src/accordions.html:57]: This is just an example to show the fixed height accordion. I didn't want to add a PF class for this example code.

**Comment by jeff-phillips-18** [comment]
@andresgalante This is straight bootstrap functionality.

**Comment by andresgalante** [comment]
@jeff-phillips-18 @rhamilto excellent! and sorry, I am studying js, I'll get there :)

**Comment by jeff-phillips-18** [comment]
@andresgalante No problem at all. I look forward to your JS contributions here and on Angular-Patternfly :)

**Pull request merged**